src/platform/datapath_raw_xdp_win.c

```diff
@@ -1023,6 +1026,8 @@ CxPlatDpRawInitialize(
         goto Error;
     }
 
+    Xdp->XdpQeoSet = (XDP_QEO_SET_FN *)Xdp->XdpApi->XdpGetRoutine(XDP_QEO_SET_FN_NAME);
```

Comment:
Should we check the result here? Or maybe just a debug assert?

Reply:
Ah, I see. You check for NUL below. I guess that's ok too, but it's probably worth a log to says "Hey, you're using an XDP that no longer has QEO"

Reply:
The error is logged in the QEO function any time QEO is attempted to be set.